src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -24,36 +25,37 @@
 using OpenTelemetry.Trace.Export;
 
 using OtlpCollector = Opentelemetry.Proto.Collector.Trace.V1;
-using OtlpTrace = Opentelemetry.Proto.Trace.V1;
 
 namespace OpenTelemetry.Exporter.OpenTelemetryProtocol
 {
     /// <summary>
-    /// The trace exporter using the OpenTelemetry protocol (OTLP).
+    /// Exporter consuming <see cref="Activity"/> and exporting the data using
+    /// the OpenTelemetry protocol (OTLP).
     /// </summary>
-    internal class TraceExporter
+    public class OtlpExporter : ActivityExporter
     {
         private readonly Channel channel;
         private readonly OtlpCollector.TraceService.TraceServiceClient traceClient;
         private readonly Metadata headers;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="TraceExporter"/> class.
+        /// Initializes a new instance of the <see cref="OtlpExporter"/> class.
         /// </summary>
         /// <param name="options">Configuration options for the exporter.</param>
-        internal TraceExporter(ExporterOptions options)
+        public OtlpExporter(OtlpExporterOptions options)
         {
             this.headers = options.Headers;
             this.channel = new Channel(options.Endpoint, options.Credentials);
             this.traceClient = new OtlpCollector.TraceService.TraceServiceClient(this.channel);
         }
 
-        internal async Task<ExportResult> ExportAsync(
-            IEnumerable<OtlpTrace.ResourceSpans> resourceSpansList,
+        /// <inheritdoc/>
+        public override async Task<ExportResult> ExportAsync(
+            IEnumerable<Activity> activityBatch,
             CancellationToken cancellationToken)
         {
             var spanExportRequest = new OtlpCollector.ExportTraceServiceRequest();
-            spanExportRequest.ResourceSpans.AddRange(resourceSpansList);
+            spanExportRequest.ResourceSpans.AddRange(activityBatch.ToOtlpResourceSpans());
```

Comment:
nit: We could remove span from this variable name.

Reply:
sure! just updated!